.vscode/settings.json

```diff
@@ -18,5 +18,11 @@
     },
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
-    "files.trimTrailingWhitespace": true
+    "files.trimTrailingWhitespace": true,
+    "mocha.options": {},
+    "mocha.requires": [
+        "ts-node/register"
+    ],
+    "mocha.files.glob": "test/**/*.js"
```

Comment:
This should probably be `"test/**/*-test.ts"`

Reply:
Cool! Do you mind explaining what this line is for?  I think it may have been auto-generated because I definitely didn't write it :)

Reply:
This is the workspace mocha configuration for the mocha test panel extension in VSCode.

`test/**/*.js` is telling it to look for files of the extension `.js` in a folder called test and all the folders underneath it. 

Our tests are written in Typescript so they have the `.ts` extension and they always follow the executable tests always follow the format of `name-test.ts`.

Glob patterns are interesting there is a great resource at https://globster.xyz/ to practice and see how they work